### PR TITLE
design the landing page and work on the categores ui

### PR DESCRIPTION
**Pull request by ashiruhammed**

### CODE REVIEW

File: frontend/src/app/page.tsx [hunk 1]
```diff
@@ -1,13 +1,275 @@
...
+      <div className="bg-white py-5 font-roboto px-4">
+        <AllFeaturesSection />
+        <FaQSection id="faqs" />
+      </div>
+      <Footer />
+    </ImageBackground>
+  );
+};
+
+const HeroSection = () => {
```
Comment (emmyduruc) [inline frontend/src/app/page.tsx]: I would put this components here into the Components directory and import them here to keep this page.tsx clean and easy to read.

move them out into component directory and export const them

File: frontend/src/app/page.tsx [hunk 2]
```diff
@@ -1,13 +1,275 @@
...
+      <div>
+        {faqs.map((faq, index) => (
+          <AccordionItem key={index} title={faq.title} content={faq.content} />
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const navigationLinks = [
```
Comment (emmyduruc) [inline frontend/src/app/page.tsx]: Create a folder name utils/constants.ts and move this in there

File: frontend/src/app/page.tsx [hunk 3]
```diff
@@ -1,13 +1,275 @@
...
+      'Yes, our platform is completely free for users. You can explore and compare different tools without any cost or commitment.',
+  },
+  {
+    title: 'Is the platform up to date with prices ?',
+    content:
+      'Yes, we regularly update our platform with the latest prices and features of various tools.',
+  },
+];
+
+const allFeatures = [
```
Comment (emmyduruc) [inline frontend/src/app/page.tsx]: Create a folder name utils/constants.ts and move this in there and export it

Comment (emmyduruc) [inline frontend/src/app/page.tsx]: Create a folder name utils/constants.ts and move this in there and export it

File: frontend/src/app/page.tsx [hunk 4]
```diff
@@ -1,13 +1,275 @@
...
+import AccordionItem from '@/components/Accordion';
+import FacebookIcon from '@/components/icon/facebook-icon';
+import InstagramIcon from '@/components/icon/instagram-icon';
+import TwitterIcon from '@/components/icon/twitter-icon';
+import ImageBackground from '@/components/ImageBackground';
+import LandingPageHeader from '@/components/LandingPageHeader';
+import Image from 'next/image';
+import { GoDotFill } from 'react-icons/go';
+
+const faqs = [
```
Comment (emmyduruc) [inline frontend/src/app/page.tsx]: Create a folder name utils/constants.ts and move this in there and export

File: frontend/src/app/layout.tsx [hunk 5]
```diff
@@ -20,10 +19,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className="flex h-screen">
+      <body>
         <QueryClientProvider client={queryClient}>
-          <Drawer />
-          <main className="flex-1 overflow-auto bg-black-100">{children}</main>
+          {/* <Drawer /> */}
```
Comment (emmyduruc) [inline frontend/src/app/layout.tsx:24]: Remove comments

File: frontend/src/app/page.tsx (hunk 3 above)
Comment (ashiruhammed) [reply frontend/src/app/page.tsx]: resolved

Comment (ashiruhammed) [reply frontend/src/app/page.tsx]: resolved

File: frontend/src/app/page.tsx (hunk 2 above)
Comment (ashiruhammed) [reply frontend/src/app/page.tsx]: resolved

File: frontend/src/app/page.tsx (hunk 1 above)
Comment (ashiruhammed) [reply frontend/src/app/page.tsx]: resolved

**Pull request merged**